fc/cli/el.py

```diff
@@ -16,7 +16,7 @@ def el():
 @click.argument('description')
 @click.option('--username')
 @click.option('--in-progress', is_flag=True)
-@click.option('--no-assign', is_flag=True)
+@click.option('--no-assign', is_flag=True, default=True)
```

Comment:
Instead of defaulting this to `True`, let's flip the entire flag to `--assign`  and plumb the changes through.  That will make it easier to understand in the future.

Same for the other one.

Reply:
Which other one?

Reply:
Sorry, the other one being `--no-assign` for triage tasks code.